bahmnicore-api/src/test/java/org/bahmni/module/bahmnicore/model/ProviderTest.java

```diff
@@ -0,0 +1,58 @@
+package org.bahmni.module.bahmnicore.model;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+public class ProviderTest {
```

Comment:
not sure utility of such test of a value object!

Reply:
This is to test equals() method only.